apps/total-typescript/src/components/app/navigation.tsx

```diff
@@ -70,38 +66,173 @@ const DesktopNav = () => {
           <FireIcon className="h-5 w-5 text-orange-400" aria-hidden="true" />
         }
       />
+      {isSellingLive && (
+        <hr
+          className="mx-3 h-1/4 w-px border-transparent bg-gray-700"
+          aria-hidden="true"
+        />
+      )}
+      {status === 'authenticated' ? (
```

Comment:
if they are `authenticated` the menu is present, otherwise if `unathenticated` show login

note that `loading` status is not accounted for and causes layout shift